keras/backend/theano_backend.py

```diff
@@ -862,11 +860,19 @@ def int_or_none(value):
     if filter_shape is not None:
         filter_shape = tuple(int_or_none(v) for v in filter_shape)
 
-    conv_out = T.nnet.conv2d(x, kernel,
-                             border_mode=th_border_mode,
-                             subsample=strides,
-                             input_shape=image_shape,
-                             filter_shape=filter_shape)
+    if filter_dilation == (1, 1):
```

Comment:
I add this if statement to support lagcy theano versions which don't support filter_dilation, not sure if it's necessary, because theano seems has [a wrapper](https://github.com/Theano/Theano/blob/a668c6c5b6d055b233aa5bc50b22800d996ffce1/theano/tensor/nnet/__init__.py#L129) to remove unrecognized arguments.